romsel_r4theme/arm9/source/fileBrowse.cpp

```diff
@@ -454,15 +488,13 @@ string browseForFile(const vector<string> extensionList, const char* username)
 					}
 				}
 				if (hasAP) {
-				dialogboxHeight = 5;
+				dialogboxHeight = 3;
 				showdialogbox = true;
-				printLargeCentered(false, 84, "Warning");
-				printSmallCentered(false, 104, "This game may not work right,");
-				printSmallCentered(false, 112, "if it's not AP-patched.");
-				printSmallCentered(false, 128, "If the game freezes, does not");
-				printSmallCentered(false, 136, "start, or doesn't seem normal,");
-				printSmallCentered(false, 144, "it needs to be AP-patched.");
-				printSmallCentered(false, 158, "B/A: OK, X: Don't show again");
+				printLargeCentered(false, 84, "Anti-Piracy Warning");
+				printSmallCentered(false, 104, "If this ROM does not have");
+				printSmallCentered(false, 112, "its Anti-Piracy patched,");
+				printSmallCentered(false, 120, "it may not work correctly!");
+				printSmallCentered(false, 142, "B: Return   A: Launch   X: Don't show again");
```

Comment:
```suggestion
				printSmallCentered(false, 142, "B: Return   A: Launch   X: Don't show again");
```
This should be reverted, because part of that text would not be shown.